styleguide/screens/Patterns/Tether/Tether.css

```diff
@@ -0,0 +1,29 @@
+.specimenContainer {
+  position: static;
+}
+
+.specimenContainer > span {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.controls {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.controls div:not(:last-of-type) {
+  margin-bottom: 1rem;
```

Comment:
We usually try to use `margin-top` to do vertical padding. Could this work?

```css
.control + .control {
  margin-top: var(--size-regular);
}
```

(It should also use the variables)